src/ProductConstructionService/ProductConstructionService.Deployment/Deployer.cs

```diff
@@ -20,25 +21,32 @@ public class Deployer
     private ContainerAppResource _containerApp;
     private readonly ResourceGroupResource _resourceGroup;
     private readonly IProcessManager _processManager;
-    private readonly IProductConstructionServiceApi _pcsClient;
+    private readonly IRedisCacheFactory _redisCacheFactory;
+    private readonly IServiceProvider _serviceProvider;
+    private readonly ILogger<Deployer> _logger;
 
     private const int SleepTimeSeconds = 20;
```

Comment:
I'd lower the sleep time to 15, maybe 10

Reply:
lowered to 10